recipes/lib-pod5/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: lib-pod5
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.21
+    - libstdcxx-ng
```

Comment:
This should be on the system, or how come that this is a runtime requirement? This should come shipped with other packages I think if it is needed by one AFAIK.

Reply:
Removed `libstdcxx-ng`.